docs/backup-and-restore.md

```diff
@@ -0,0 +1,73 @@
+Ozone includes backup and restore out of the box. The backup and restore is powered by the powerful and popular [Restic](https://restic.readthedocs.io/en/stable/)  project. Meaning all the backup [backends](https://restic.readthedocs.io/en/stable/030_preparing_a_new_repo.html) supported by Restic can be supported by Ozone.
+
+We wrap Restic in a convience project to allow it to seamlessly intergrate with Ozone and if you are curious about the details you can take a look at the wrapper [project](https://github.com/mekomsolutions/restic-compose-backup)
+
+The backup and restore process is broken into two different services;
+
+- Backup Service (Enabled by default)
+- The restore service, which can be run on time to restore from  a backup generated by the backup service
+
+###  Backup and Restore when using the Ozone helper scripts
+These instructions assume you are already in the Ozone Scripts directory --> see the start Documentation
+#### Backup to Local Path
+```
+export BACKUP_PATH=/tmp/backup
+```
+`./start.sh`
```

Comment:
How come it's always `./start.sh` for every action?

Reply:
It is controlled by `export RESTORE="true"`. In both cases, you are starting  Ozone it's just if you starting from backup or not. This way we reuse the existing Ozone scripts without adding more scripts.

Reply:
> Restic supporting a specific envvar does not mean that Ozone will support it automatically? There is additional work to be done in Ozone for that?

Yes we need to make docker compose changes so the env var is passed to the Restic container

Reply:
@rbuisson the use of `start.sh` is not documented anywhere. I did a search in your large PR and even there it isn't mentioned. Why is that, and what's the difference with `start-demo.sh`?